channel/consensus_channel/consensus_channel.go

```diff
@@ -107,6 +113,24 @@ func (c *ConsensusChannel) Includes(g Guarantee) bool {
 	return c.current.Outcome.includes(g)
 }
 
+// IncludesTarget returns whether or not the consensus state includes a guarantee targeting the given channel
+func (c *ConsensusChannel) IncludesTarget(target types.Destination) bool {
+	return c.current.Outcome.includesTarget(target)
+}
```

Comment:
Is this check thorough enough? Is there an attack where a guarantee is added with (say) a zero amount?

Reply:
`IncludesTarget` isn't used to determine if we sign a state so I don't think it needs to be concerned with safety. `IncludesTarget` is just used to check the if the state we've already signed includes a guarantee.

The proposal system in the consensus channel should enforce that we only sign a state with the specific outcome we expect.